modelator/const_values.py

```diff
@@ -39,6 +39,10 @@ def apalache_release_url(expected_version):
     return f"https://github.com/informalsystems/apalache/releases/download/v{expected_version}/apalache.zip"
 
 
+def apalache_checksum_url(expected_version: str):
```

Comment:
We should do some sanity check on the argument, which is originally written by the user. The same for the above function.

Reply:
Very good question. I was also thinking to sanitize the argument. Probably a check at the CLI method?

Reply:
yeah, that's a very good comment!